src/lio/os/file.c

```diff
@@ -1128,7 +1128,7 @@ int va_timestamp_set_attr(lio_os_virtual_attr_t *va, lio_object_service_fn_t *os
     n = (int)(long)va->priv;  //** HACKERY ** to get the attribute prefix length
     key = &(fullkey[n+1]);
 
-    if (strlen(fullkey) < n) {  //** Nothing to do so return;
+    if ((int)strlen(fullkey) < n) {  //** Nothing to do so return;
```

Comment:
Elaborate? What are you looking for? I did look at all of them each 
individually and didn't see any issues.  Should we document each warning 
as well?  IF so it should be in the guide.